exporter/googlecloudpubsubexporter/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlecloudpubsubexporter
 
-go 1.22.0
+go 1.22.7
+
+toolchain go1.22.12
```

Comment:
Same comment in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37659

Reply:
no longer an issue after https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37875